base/Word/sqrt.kind

```diff
@@ -0,0 +1,9 @@
+Word.sqrt<size: Nat>(word: Word(size)): Word(size)
```

Comment:
there's a bug hiding somewhere here, because the function is returning 1 less than it should for perfect squares
```
Test: String
  let nat_list = List.range(25)
  let to_sqrt = (n) ret = Nat.to_u32(n)
                open ret
                root_word = Word.sqrt!(ret.value)
                Word.to_nat!(root_word)
  root_list = List.map!!(to_sqrt, nat_list)
  show = List.show!(Pair.show!!(Nat.show, Nat.show))

  show(List.zip!!(nat_list, root_list))
```

Reply:
This is now fixed in c5d427eca1ebad826a14346cdd712b398d36e079 and works now too for odd bits sizes like e.g. `Word(5)`.